src/components/vm/VmInitializer.tsx

```diff
@@ -202,6 +226,7 @@ export default function VmInitializer() {
       requestSignMessage,
       vmNear: near,
       signedIn,
+      idOS,
```

Comment:
After idOS initialization we are storing it in our `useAuthStore` for easy access to idOS methods